engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/schema/GraphQLFieldInfo.kt

```diff
@@ -0,0 +1,11 @@
+package graphql.nadel.enginekt.schema
+
+import graphql.nadel.OperationKind
+import graphql.nadel.Service
+import graphql.schema.GraphQLFieldDefinition
+
+data class GraphQLFieldInfo(
```

Comment:
I would expect that we have a type/kind on it which tells me if it is a renamed field, hydration etc. Also: if we have field definition we should have both definitions, but then again we should try to make it self containted wiithout schema referenced if we only need s smal subset.